Files/Views/LayoutModes/GridViewBrowser.xaml.cs

```diff
@@ -412,15 +412,23 @@ private void FolderSettings_GridViewSizeChangeRequested(object sender, EventArgs
         private async void ReloadItemIcons()
         {
             ParentShellPageInstance.FilesystemViewModel.CancelExtendedPropertiesLoading();
-            foreach (ListedItem listedItem in ParentShellPageInstance.FilesystemViewModel.FilesAndFolders.ToList())
+
+            try
             {
-                listedItem.ItemPropertiesInitialized = false;
-                if (FileList.ContainerFromItem(listedItem) != null)
+                foreach (ListedItem listedItem in ParentShellPageInstance.FilesystemViewModel.FilesAndFolders.ToList())
```

Comment:
I added `ToList()` here in order to solve the same issue. Have you experienced this bug despite it?

Reply:
@gave92 I haven't, the bug report was from v1.3 so it could be your fix was enough.

Reply:
Ok I don't have the confirmation my fix works either. But perhaps we can try with it first.